rasa/core/broker.py

```diff
@@ -56,24 +60,109 @@ def publish(self, event: Dict[Text, Any]) -> None:
         raise NotImplementedError("Event broker must implement the `publish` method.")
 
 
+# noinspection PyUnresolvedReferences
+def initialise_pika_connection(
+    host: Text,
+    username: Text,
+    password: Text,
+    connection_attempts: int = 20,
+    retry_delay_in_seconds: Union[int, float] = 5,
+) -> "BlockingConnection":
+    """Create a Pika `BlockingConnection`.
+
+    Args:
+        host: Pika host
+        username: username for authentication with Pika host
+        password: password for authentication with Pika host
+        connection_attempts: number of connection attempts before giving up
+        retry_delay_in_seconds: delay in seconds between connection attempts
+
+    Returns:
+        Pika `BlockingConnection` with provided parameters
+    """
+
+    import pika
+
+    parameters = pika.ConnectionParameters(
+        host,
+        credentials=pika.PlainCredentials(username, password),
+        connection_attempts=connection_attempts,
+        # Wait between retries since
+        # it can take some time until
+        # RabbitMQ comes up.
+        retry_delay=retry_delay_in_seconds,
+    )
+    return pika.BlockingConnection(parameters)
+
+
+# noinspection PyUnresolvedReferences
```

Comment:
what do we need these for?

Reply:
thought it was needed for the type checking without import, but that's not the case 👍